ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -111,6 +119,14 @@ public struct Avatar: View, TokenizedControlView {
                                                                                                            secondaryText: state.secondaryText))
         let shouldUseCalculatedColors = !initialsString.isEmpty && style != .overflow
 
+        let activityImage: Image = {
+            if let image = state.activityImage {
+                return Image(uiImage: image)
+            } else {
+                return Image("ic_fluent_presence_unknown_10_regular", bundle: FluentUIFramework.resourceBundle)
```

Comment:
Should we even have a default activity image? Seems like it might be better to just not display anything if they haven't set something.

Reply:
you are inside fluent library  at this point. do you really need to specify the bundle name?

Reply:
actually.... why can't activityImage can't be just nil? we only need image when we need impage. we shouldn't every show unknown image